packages/frontend/src/components/common/AnchorLink/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { AnchorButton } from '@blueprintjs/core';
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+
+const AnchorLink: FC<
+    { href: string } & React.ComponentProps<typeof AnchorButton>
```

Comment:
I think we can remove the properties from the AnchorButton since we are not using that component.